pkg/kine/drivers/sqlite/sqlite_test.go

```diff
@@ -51,7 +53,13 @@ func TestMigration(t *testing.T) {
 	}
 
 	ctx := context.Background()
-	if _, err := sqlite.New(ctx, dbPath); err != nil {
+	connPoolConfig := generic.ConnectionPoolConfig{
+		MaxIdle:     5,
+		MaxOpen:     5,
+		MaxLifetime: 60 * time.Second,
+		MaxIdleTime: 60 * time.Second,
```

Comment:
In the CLI, the default for `MaxIdleTime` is `0*time.Second`. Can we use that here as well?

Reply:
Yes, good catch!